__tests__/app.test.js

```diff
@@ -38,25 +51,44 @@ describe('GET /api/topics', () => {
 
 });
 
-describe('GET /api', () => {
-    test('GET 200 /api', () => {
+describe('/api/articles', () => {
+    test('GET 200 /api/articles/1', () => {
         return request(app)
-        .get('/api')
-        .expect(200)
-        .then(({body})=>{
-            expect(body).toEqual(endpointsData)
-            expect(typeof body).toBe('object')
-        })
+            .get('/api/articles/1')
+            .expect(200)
+            .then(({ body: { articles } }) => {
+                const article = articles[0]
+                expect(article.article_id).toBe(1)
+                expect(article.title).toBe("Living in the shadow of a great man")
+                expect(article.topic).toBe("mitch")
+                expect(article.author).toBe("butter_bridge")
+                expect(article.body).toBe("I find this existence challenging")
+                expect(article.created_at).toBe("2020-07-09T20:11:00.000Z")
+                expect(article.votes).toBe(100)
+                expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+
```

Comment:
When you make a request to /api/articles/1, shouldn't the return be an object, not an array?